packages/serverpod/lib/src/server/features.dart

```diff
@@ -39,9 +39,9 @@ class Features {
 
   /// Returns true if the web server is enabled.
   static bool get enablePersistentLogging =>
-      _instance._config.sessionLogs?.persistentEnabled == true;
+      _instance._config.sessionLogs.persistentEnabled == true;
```

Comment:
```suggestion
      _instance._config.sessionLogs.persistentEnabled;
```

Since there is no `null` in the chain anymore.